config/state_machine/moj.yml

```diff
@@ -1263,6 +1264,7 @@ case_types:
                 assign_to_new_team:
                 destroy_case:
                 edit_case:
+                flag_for_clearance:
```

Comment:
What is this action being used for, since these cases are always flagged for clearance?